bin/incredible-squaring-avs/src/commands/avs/mod.rs

```diff
@@ -516,92 +679,289 @@ impl<Ext: clap::Args + fmt::Debug + Send + Sync + 'static> AvsCommand<Ext> {
 
 #[allow(clippy::too_many_arguments)]
 /// Register operator in eigenlayer and avs
-pub async fn register_operator_with_el_and_avs(
+pub async fn register_operator_with_el_and_deposit_tokens_in_strategy(
+    metadata_uri: String,
+    allocation_delay: u32,
     operator_pvt_key: Option<String>,
     rpc_url: String,
     ecdsa_keystore_path: String,
     ecdsa_keystore_password: String,
+    permission_controller_address: Address,
+    rewards_coordinator: Address,
+    allocation_manager: Address,
     registry_coordinator_address: Address,
-    operator_state_retriever_address: Address,
     delegation_manager_address: Address,
     avs_directory_address: Address,
     strategy_manager_address: Address,
     erc20_strategy_address: Address,
-    bls_keystore_path: &str,
-    bls_keystore_password: &str,
-    operator_to_avs_registration_sig_salt: FixedBytes<32>,
-    operator_to_avs_registration_sig_expiry: U256,
-    quorum_numbers: Bytes,
-    socket: String,
     deposit_tokens: U256,
 ) -> eyre::Result<()> {
     let signer;
+    dbg!(operator_pvt_key.clone());
```

Comment:
There are some `dbg!`s left in the code

Reply:
removed . should i also remove from integration tests?

Reply:
I think we can leave those